### ISSUE TITLE
Make it work with OKD

### ISSUE THREAD

**Issue by fabiand** [openshift/assisted-installer#352]
The docs should contain a hint of how AI can be used to deploy OKD instead of OCP

**Comment by eranco74** [comment]
That is correct, but the issue is relevant for [assisted-service](https://github.com/openshift/assisted-service).
This repo and the docs in it deals with the installer running on the live-iso.
The assisted-service holds the full deployment and docs of the assisted-installer service.
Moving the issue there
